src/setup-beam.js

```diff
@@ -37,7 +42,7 @@ async function main() {
   await maybeInstallRebar3(rebar3Spec)
 }
 
-async function installOTP(otpSpec, osVersion) {
+async function maybeInstallOTP(otpSpec, osVersion) {
```

Comment:
Function name should not be `maybe...`. Execution is already made conditional by external factors.

Reply:
Ah yes I was going back and forth with where to put the conditional. Will update.